documentation/docs/how_tos/outputs_and_transactions/01_send_transaction.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: Send a Transaction
+description: 'How to send a value transaction using wallet.rs'
+image: /img/logo/iota_mark_light.png
+keywords:
+- how to
+- java
+- nodejs
+- python
+- rust
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import NodejsCode from "../../libraries/nodejs/how_to/outputs_and_transactions/_05_send_transaction/_code_sample.mdx";
+import NodejsOutput from "../../libraries/nodejs/how_to/outputs_and_transactions/_05_send_transaction/_expected_output.md";
+import PythonCode from "../../_admonitions/_guide_coming_soon.md";
```

Comment:
We have a transaction example for python https://github.com/iotaledger/wallet.rs/blob/develop/bindings/python/native/examples/3-send-transaction.py

Reply:
yeah, it seems I missed it when adding the links.  Updated! Thanks for catching this!